src/apps/loyalty/server/enroll_loyalty_applicant.ts

```diff
@@ -0,0 +1,18 @@
+import fetch from "node-fetch"
+
+export function fetchCollectorProfile(accessToken: string): any {
```

Comment:
By adding that, and removing the `any` - it will run the inference correctly:

![screen shot 2017-04-10 at 20 15 04](https://cloud.githubusercontent.com/assets/49038/24887545/718cebb2-1e2a-11e7-93a7-0570fe18366f.png)

(looks like I meant `cmd`, not `alt` above too)